cli/ceph/orch/tuned_profile.py

```diff
@@ -16,7 +16,7 @@ def apply(self, spec_file):
             spec_file: tune spec file details
         """
         cmd = f"{self.base_cmd} apply -i {spec_file}"
-        out = self.execute(sudo=True, cmd=cmd)
+        out = self.execute(sudo=True, cmd=cmd, check_ec=True)
```

Comment:
Don't keep this as constant value. Get it from user.

Reply:
Thanks @vamahaja
Updated